src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <ol class="calendar">
```

Comment:
use of` divs `and `container` will be better here instead of `list`
also use modifier classes for container `div`

Reply:
done.